src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -12,6 +12,9 @@
 import org.testng.annotations.Test;
 
 import java.io.File;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
 
 public class JBWAIntegrationTest extends BaseTest {
```

Comment:
A bit nitpicking here: should the name be `UnitTest` rather than integration test? I though integration test here means something different (at least in GATK3)?